src/change-management/change-management.rst

```diff
@@ -0,0 +1,366 @@
+problemi:
+
+-  Una nuova versione di ANPR viene rilasciata inizialmente solo negli
+   ambienti di test e pre-subentro, dove vi rimane per almeno 15 giorni.
+
+-  Trascorsi i 15 giorni, se non vi sono segnalazioni aperte che
+   riportino gravi problemi, la versione viene promossa alla fase di
+   collaudo del Ministero e quindi in produzione.
+
+Descrizione della versione
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Per garantire compatibilità durante l'aggiornamento è necessario avere
+una descrizione puntuale della versione corrente di ANPR, accessibile in
+modo programmatico.
+
+A tal proposito è possibile utilizzare il servizio ``Status`` per:
+
+-  richiedere la versione corrente dell'istanza di ANPR utilizzata
+   chiamando il metodo ``GetVersion``. Questo metodo ritorna la versione
+   completa (major, minor, patch).
+
+-  richiedere tutte le versioni major supportate chiamando il metodo
+   ``GetSupportedVersions`` .
+
+I WSDL di quella versione saranno disponibili su GitHub all'indirizzo
+https://github.com/italia/anpr/wsdl/ oppure su
+https://www.anpr.interno.it/portale/documentazione-tecnica.
+
+È inoltre possibile consultare l'elenco delle versioni supportate in
+tutti gli ambienti nella seguente pagina: https://docs.anpr.it/status
+(Nota: La pagina al momento è accessibile poiché il processo di change
+management è in fase di implementazione).
+
+Nota: Nel caso della web application, si assume che sia associata a una
+specifica versione delle API dei webservices di backend, e verrà
+indicato chiaramente nel footer la versione (in ambiente test e test
+comuni anche con un link al changelog).
+
+Storico delle versioni
+~~~~~~~~~~~~~~~~~~~~~~
+
+Lo storico delle versioni permette di rendere facilmente consultabili i
+cambiamenti introdotti da una nuova versione di ANPR. Lo storico delle
+versioni viene implementato tramite `ChangeLog <changelog>`__ : un file
+di testo contenente un sommario dei cambiamenti presenti in ogni
+versione di ANPR.
+
+È possibile consultare lo storico delle versioni all'indirizzo:
+https://docs.anpr.it/changelog (Nota: La pagina al momento è accessibile
+poiché il processo di change management è in fase di implementazione.)
+
+Al rilascio di una nuova versione lo storico dei cambiamenti della
+versione stessa verrà anche pubblicato nella newsletter (nome newsletter
+ancora da definire).
+
+Il ChangeLog viene aggiornato ogni qual volta si rilascia una nuova
+versione di ANPR. Il file è diviso in varie sezioni, una sezione per
+versione, e le sezioni seguono un ordine cronologico inverso per
+facilitare la lettura dei cambiamenti più recenti.
+
+Ogni sezione del ChangeLog deve contenere:
+
+-  La versione di ANPR a cui si riferisce
+
+-  La data di rilascio
+
+-  Una lista dei cambiamenti più importanti contenuti nella versione.
+
+Se la nuova versione contiene un cambiamento relativo a una issue
+segnalata nell'issue tracker, l'url della issue deve essere incluso nel
+sommario.
+
+Esempio di ChangeLog ANPR:
```

Comment:
Qui si potrebbe usare il box "esempio" messo a disposizione da Docs Italia => https://docs.italia.it/italia/docs-italia/docs-italia-guide/it/bozza/appendice-2/box.html